src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -209,6 +209,21 @@ bool java_bytecode_languaget::typecheck(
       return true;
   }
 
+  // find and mark all implicitly generic class types
+  // this can only be done once all the class symbols have been created
+  for(auto c : java_class_loader.class_map)
+  {
+    if(c.second.parsed_class.name.empty())
+      continue;
+    try
+    {
+      mark_java_implicitly_generic_class_type(c.first, symbol_table);
+    }
+    catch(missing_outer_class_symbol_exceptiont &)
+    {
```

Comment:
Consider printing a warning (perhaps later downgraded to debug message) when this bails out